modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-loki-tech-preview_{context}"]
+:FeatureName: Loki Operator
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-loki"]
+= About the Loki Operator
+[role="_abstract"]
+Loki is currently offered as an alternative to Elasticsearch as a log store for the {logging}.
+
+= Deploying the Loki Collector
+
+.Deploy loki-operator under openshift-operators namespace
+[source, terminal]
+----
+oc create -f  aosqe-tools/logging/log_template/loki-operator/loki-og-sub.yaml
+----
+
+.Set Environment variables
+[source, yaml]
+----
+export NAMESPACE=openshift-logging
+export SECRETNAME=credsecret
+export AWS_DEFAULT_REGION=us-east-2
+export LOKI_BUCKET_NAME="aosqe-loki-bucket"
+export ENDPOINT="https://s3.${REGION}.amazonaws.com"
+export AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY_ID>
+export AWS_SECRET_ACCESS_KEY=<AWS_SECRET_ACCESS_KEY>
+export STORAGE_CLASS=<storage_class>
+export LOKI_BUCKET_NAME=loki-log54
+----
+
+.Create s3 bucket
+[source, terminal]
+----
+aws s3api create-bucket --bucket $LOKI_BUCKET_NAME --region $AWS_DEFAULT_REGION
+aws s3api create-bucket --bucket $LOKI_BUCKET_NAME --region $AWS_DEFAULT_REGION --create-bucket-configuration LocationConstraint=$AWS_DEFAULT_REGION
+----
+
+.Create Storage Secret
+[source, terminal]
+----
+oc -n "${NAMESPACE}" delete secret ${SECRETNAME}
+oc -n "${NAMESPACE}" create secret generic ${SECRETNAME} \
+    --from-literal=endpoint=${ENDPOINT} \
+    --from-literal=region=${REGION} \
+    --from-literal=bucketnames=${LOKI_BUCKET_NAME} \
+    --from-literal=AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \
+    --from-literal=access_key_secret=${AWS_SECRET_ACCESS_KEY}
+----
+
+.Deploy lokistack under openshift-logging namespace
+[source, terminal]
+----
+oc process -f aosqe-tools/logging/log_template/loki-operator/lokistack_s3_template.yaml  -p STORAGE_CLASS=${STORAGE_CLASS} -p SECRET_NAME=${SECRETNAME} |oc create -f -
+----
```

Comment:
I think we should be displaying the components that are created as a part of the mentioned LokiStack above.

$ oc get pods -n openshift-logging
NAME                                           READY   STATUS    RESTARTS   AGE
lokistack-dev-compactor-0                      1/1     Running   0          63m
lokistack-dev-distributor-8696c45889-slb9k     1/1     Running   0          63m
lokistack-dev-gateway-5bfc4bdd67-snjjl         2/2     Running   0          63m
lokistack-dev-index-gateway-0                  1/1     Running   0          63m
lokistack-dev-ingester-0                       1/1     Running   0          63m
lokistack-dev-querier-9ccd98f9d-f6c5k          1/1     Running   0          63m
lokistack-dev-query-frontend-58bc479b7-dp8h9   1/1     Running   0          63m